test/format/wts.c

```diff
@@ -466,6 +467,12 @@ create_database(const char *home, WT_CONNECTION **connp)
     if (GV(CACHE_EVICTION_DIRTY_TRIGGER) != 0)
         CONFIG_APPEND(p, ",eviction_dirty_trigger=%" PRIu32, GV(CACHE_EVICTION_DIRTY_TRIGGER));
 
+    /* Eviction (updates) configuration. */
+    if (GV(CACHE_EVICTION_UPDATES_TARGET) != 0)
```

Comment:
My understanding is that CACHE_EVICTION_UPDATES_TARGET has to be above CACHE_EVICTION_DIRTY_TRIGGER. Could a test/format config randomly violate that rule?

Reply:
I think you meant that `CACHE_EVICTION_DIRTY_TRIGGER` should be greater than `CACHE_EVICTION_UPDATES_TARGET`. In test/format, these thresholds are never randomly generated; they're either explicitly set by the user or configured for specific edge cases within the test itself. Otherwise, defaults are used.